src/plugins/intel_npu/src/plugin/npuw/partitioning/online/snapshot.cpp

```diff
@@ -66,7 +66,7 @@ void Snapshot::buildGraph() {
 
         auto nh = m_graph->create();
         auto group = std::make_shared<Group>(ov_node, gid, nh, m_graph, shared_from_this());
-        m_graph->meta(nh).set(group);
+        m_graph->meta(std::move(nh)).set(group);
```

Comment:
please don't do this. asking meta for a node handle doesn't take ownership over the node handle.